### PR TITLE
WIP: Try to add react-native-image-crop-picker

### PR DESCRIPTION
**Pull request by forki**
Having trouble to create Fable.Import.ReactNativeImageCropPicker.fs

### CODE REVIEW

File: src/Fable.Import.ReactNativeImageCropPicker.fs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+module ReactImageCropPicker =
+
+    type CameraStatic =
+        inherit React.ComponentClass<CameraProperties>
+
+    and CameraProperties = obj
+
+    type Globals =
+        [<Import("default", from="react-native-image-crop-picker")>]
```
Comment (forki) [inline src/Fable.Import.ReactNativeImageCropPicker.fs]: this is complete nonsense - how did you do that binding?

**Comment by forki** [comment]
image

that's where I'm hanging right now

**Comment by forki** [comment]
ok seems the touchablehighlight doesn't work yet.

I used the onpress of the hello world text and now I get:

image

**Comment by forki** [comment]
ok imagepicker is opening.

now I need to find out how to process the result obj

**Comment by Krzysztof-Cieslak** [comment]
fable-imports can be added as devDeps not normal deps.

**Comment by forki** [comment]
done

**Comment by forki** [comment]
merging this changed F# amount to 17% ;-)

**Comment by forki** [comment]
selectimage

**Comment by forki** [comment]
API looks much clearer now: https://github.com/Krzysztof-Cieslak/fable-react_native-demo/blob/master/src/ImagePickerApp.fs#L30

**Pull request merged**